app/src/main/java/net/gsantner/opoc/frontend/base/GsFragmentBase.java

```diff
@@ -154,13 +159,26 @@ protected void attachToolbarClickListenersToFragment() {
         }
     }
 
+    public void post(final Runnable action) {
```

Comment:
Experimental: Ability to post to fragments. Very useful as child fragments are resumed _after_ parent activities.